instaclustr/structs.go

```diff
@@ -74,12 +74,13 @@ type CreateRequest struct {
 	ClusterName           string          `json:"clusterName"`
 	Bundles               []Bundle        `json:"bundles"`
 	Provider              ClusterProvider `json:"provider"`
-	SlaTier               string          `json:"slaTier"`
+	SlaTier               string          `json:"slaTier,omitempty"`
 	NodeSize              string          `json:"nodeSize"`
-	DataCentre            string          `json:"dataCentre"`
-	ClusterNetwork        string          `json:"clusterNetwork"`
-	PrivateNetworkCluster string          `json:"privateNetworkCluster"`
-	PCICompliantCluster   string          `json:"pciCompliantCluster"`
+	DataCentre            string          `json:"dataCentre,omitempty"`
+	DataCentres           []DataCentre    `json:"dataCentres,omitempty"`
```

Comment:
Not directly related to this line but I can't find an appropriate place to put this comment: don't we have to update some documentation in this repo as well?

Reply:
good point.